src/serialisation.rs

```diff
@@ -6,26 +6,32 @@
 
 use crate::commitment_scheme::kzg10::Commitment;
 use crate::fft::{EvaluationDomain, Evaluations, Polynomial};
-use anyhow::{Error, Result};
 use dusk_bls12_381::{BlsScalar, G1Affine, G2Affine};
 use dusk_bytes::{DeserializableSlice, Serializable};
-use thiserror::Error;
 
 /// Defines all of the possible Serialisation errors
-#[derive(Error, Debug)]
+#[derive(core::fmt::Debug)]
 pub enum SerialisationErrors {
-    #[error("There are not enough bytes to perform deserialisation")]
     NotEnoughBytes,
-    #[error("Cannot decompress point, as it is not in a canonical format")]
     PointMalformed,
-    #[error("Cannot deserialise scalar, as it is not in a canonical format")]
     BlsScalarMalformed,
 }
 
+// XXX: Should be covered with a conditional compilation flag
```

Comment:
You can introduce the `std` flag already and make it default. 
And mask this `Display` impls directly inside.

Reply:
Gotcha :)